lib/Runtime/Language/JavascriptStackWalker.cpp

```diff
@@ -444,7 +444,7 @@ namespace Js
                 inlineeOffset = inlinedFrameWalker.GetCurrentInlineeOffset();
             }
         }
-        else if (ScriptFunction::Is(parentFunction) && HasInlinedFramesOnStack())
+        else if (ScriptFunction::Test(parentFunction) && HasInlinedFramesOnStack())
```

Comment:
@kunalspathak that's a good point. I just rebased the code (3 months old :) ) to master. I will rescan and see what else I might do it here.